### PR TITLE
Add ICRA 40th anniversary, ROScon jp and ROScon ch

### PR DESCRIPTION
**Pull request by knmcguire**
This adds the ICRA 40th anniversary, ROScon japan and ROScon china.

ROScon germany has been confirmed but there is no website yet

### CODE REVIEW

File: events.yaml [hunk 1]
```diff
@@ -114,6 +129,14 @@
   country: Switzerland
   category: Competition
 
+- title: ROSCon China 2024
+  link: http://roscon.roseducation.org.cn/2024/en/
+  start_date: 2024-11-2
+  end_date: 2024-11-3
+  city: Odense
```
Comment (msadowski) [inline events.yaml:136]: Thanks Kimberly! I'm on mobile so can't suggest the changes reliably but these fields seem to be from Roscon 2024?

**Review by msadowski — CHANGES_REQUESTED** [review]

File: events.yaml (hunk 1 above)
Comment (knmcguire) [reply events.yaml:136]: ah woops, classic copy past. I'll fix it, thanks!

**Comment by knmcguire** [comment]
I've also added ROScon spain to it as well along with fixing ROScon china. Lot of ROScons this year!

**Comment by knmcguire** [comment]
Just let me know if you are okay with this and I'll merge:)

**Comment by msadowski** [comment]
Merged, sorry for the delay - busy days!

**Pull request merged**